src/EditorFeatures/Core/Shared/Extensions/ITextBufferEditExtensions.cs

```diff
@@ -28,8 +22,6 @@ public static ITextSnapshot ApplyAndLogExceptions(this ITextBufferEdit edit)
             }
             catch (Exception e) when (ErrorReporting.FatalError.ReportWithoutCrash(e))
             {
-                s_lastException = e;
```

Comment:
this looks like it shoudl be kept right?  it was for crash dump investigations it looks like.

Reply:
Sure, I can restore.